autoload/lsp/utils.vim

```diff
@@ -2,6 +2,13 @@ function! lsp#utils#is_remote_uri(uri) abort
     return a:uri =~# '^\w\+::' || a:uri =~# '^\w\+://'
 endfunction
 
+function! s:decode_uri(uri) abort
+    let ret = a:uri
```

Comment:
Use l:ret to match with existing coding standards

Reply:
Done, please check.